lib/src/Dscp/Educator/Launcher/Mode.hs

```diff
@@ -66,11 +67,11 @@ type CombinedWorkMode ctx m =
 -- TODO Separate resources and non-resources.
 data EducatorContext = EducatorContext
     {
-      _ecParams     :: EducatorParams
-    , _ecDB         :: SQLiteDB
-    , _ecSecret     :: EducatorSecret
+      _ecParams     :: !EducatorParams
+    , _ecDB         :: !SQLiteDB
+    , _ecKeys       :: !(KeyResources EducatorNode)
```

Comment:
I wanted to change it many times before but tried to avoid conflicts, now is the time. Please take a look at witness mode context -- it essentially consists of three things: parameters, resources and variables. SQLiteDB, KeyResources are Resources, so please put EducatorResources directly into EducatorContext and fix related HasLens instances.